ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master-presubmits.yaml

```diff
@@ -1453,7 +1526,6 @@ presubmits:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-ingress-operator-master-okd-scos-images
-    optional: true
```

Comment:
I didn't change okd-scos-images in `ci-operator/config` not sure why `make jobs` made the changes.

Reply:
Probably https://issues.redhat.com/browse/DPTP-4313.